drivers/ethernet/eth_mcux.c

```diff
@@ -897,7 +912,7 @@ static int eth_0_init(struct device *dev)
 	/* handle PHY setup after SMI initialization */
 	eth_mcux_phy_setup(context);
 
-	LOG_DBG("MAC %02x:%02x:%02x:%02x:%02x:%02x",
+	LOG_DBG("[%p] MAC %02x:%02x:%02x:%02x:%02x:%02x", context->base,
```

Comment:
I would prefer printing DT_ETH_MCUX_x_NAME or instance number rather than the address in context->base.

Reply:
I'll do that.

Reply:
I changed it in 99237a1472faf2daf552a768ce18daf904a06f3f